DOCS-7570-token-duration (#133)
mulesoft/docs-access-management: modules/ROOT/pages/connected-apps-developers.adoc

When a developer creates an app, the app requests access to user data.
When an end user receives such a request and authorizes the app to access their data, the app receives an access token, which allows the app to call Anypoint Platform on the user’s behalf.

> The access token allows the app to see who the user is by using the user’s Anypoint Platform credentials. The duration of a token is 90 minutes.

Reviewer: ```suggestion
The access token allows the app to see who the user is by using the user’s Anypoint Platform credentials. A token is valid for up to 90 minutes.
```
